app/experimenter/static/core/package.json

```diff
@@ -44,28 +44,7 @@
     "react-select": "3.1.0"
   },
   "devDependencies": {
```

Comment:
Random thought, but this feels like a possibly odd place for a package.json file since it gets served at https://experimenter.services.mozilla.com/static/core/package.json

Reply:
in general i dont think any of these should have lived in the `static` folder but stuff was already there so i was trying to minimize how much i reorganized files. i think the typical pattern is to create a source directory for all the front end code that django never sees and add the output folder for the webpack/parcel build as a static directory in django. i can go do that work if anyone wants.